docs/guide/scala_guide.md

```diff
@@ -31,19 +31,13 @@ Scala 2 and 3 are both supported.
 
 ## Install
 
-To add a dependency on Fory scala for scala 2 with sbt, use the following:
+To add a dependency on Fory scala for with sbt, use the following:
 
 ```sbt
-libraryDependencies += "org.apache.fory" % "fory-scala_2.13" % "0.11.1"
+libraryDependencies += "org.apache.fory" %% "fory-scala" % "0.11.1"
```

Comment:
note the `%%` syntax - means sbt will append the appropriate scala version to the jar name for you